src/app/hooks/useWalletSession.ts

```diff
@@ -12,24 +14,17 @@ const useWalletSession = () => {
 
   const setSessionStartTime = () => {
     const sessionStartTime = new Date().getTime();
-    chrome.storage.session.set({ [SESSION_START_TIME_KEY]: sessionStartTime });
+    setSessionItem(SESSION_START_TIME_KEY, sessionStartTime);
   };
 
   const clearSessionTime = async () => {
-    await chrome.storage.session.remove(SESSION_START_TIME_KEY);
-  };
-
-  const getSessionStartTime = async () => {
-    const { sessionStartTime } = await chrome.storage.session.get(SESSION_START_TIME_KEY);
-    return sessionStartTime;
+    await removeSessionItem(SESSION_START_TIME_KEY);
   };
 
   const shouldLock = async () => {
-    const pHash = await chrome.storage.session.get('pHash');
-
+    const pHash = await getSessionItem(SeedVaultStorageKeys.PASSWORD_HASH);
```

Comment:
why isn't this in the SeedVault class?

Reply:
moved the behavior to the seedvault as `isVaultUnlocked`